code/modules/cargo/packs.dm

```diff
@@ -2182,6 +2219,55 @@
 	crate_name = "art supply crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
+	access = ACCESS_CAPTAIN
+	cost = 25000
+	contains = list(/obj/item/pen/fountain/captain)
+	crate_name = "captain pen"
+	crate_type = /obj/structure/closet/crate/secure/weapon //It is a combat pen
+
+/datum/supply_pack/misc/captain_headset
+	name = "Captain Headset Key"
+	desc = "A spare Captain headset key, just in case The Captain ''lost'' it again." 
+	access = ACCESS_CAPTAIN
+	cost = 20000
+	contains = list(/obj/item/encryptionkey/heads/captain)
+	crate_name = "captain headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/cargo_headsets
+	name = "Cargo Headset Keys"
+	desc = "Five Headset Keys for Cargo Techs. To help yell at miners for more plasma to sell."
+	access = ACCESS_CARGO
+	cost = 2500
+	contains = list(/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo)
+	crate_name = "cargo headset keys"
+	crate_type = /obj/structure/closet/crate/secure
+
+/datum/supply_pack/misc/ce_headset
+	name = "Chief Engineer Headset Key"
+	desc = "A spare CE headset key. For when the CE throws there key into the SM again."
+	access = ACCESS_CE
+	cost = 10000
+	contains = list(/obj/item/encryptionkey/heads/ce)
+	crate_name = "chief engineer headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/cmo_headset
```

Comment:
Why should head access keys be purchasable?

Reply:
For when the CMO headset keys are all stolen by Robots